### PR TITLE
docs: Node.js setup instructions

### PR DESCRIPTION
**Pull request by vasfvitor**
Wrote this aiming #1864 

After we shape this I'll open a PR to the dev branch

page is : "guides/prerequisites"

### CODE REVIEW

File: src/content/docs/guides/prerequisites/index.mdx [hunk 1]
```diff
@@ -194,6 +194,28 @@ Be sure to restart your Terminal (and in some cases your system) for the changes
...
+```sh
+node -v 
+# v20.10.0
+npm -v 
+# 10.2.3
+```
+
+Be sure to restart your Terminal (and in some cases your system) for the changes to take affect.
+
+While npm is the default package manager, you can easily enable pnpm and yarn by running `corepack enable` on your Terminal.
```
Comment (dreyfus92) [inline src/content/docs/guides/prerequisites/index.mdx]:
```suggestion
While npm is the default package manager for Node.js, you can also use others like pnpm or yarn. To enable these, run `corepack enable` in your Terminal. This step is optional and only needed if you prefer using a package manager other than npm.your Terminal.
```

File: src/content/docs/guides/prerequisites/index.mdx [hunk 2]
```diff
@@ -194,6 +194,28 @@ Be sure to restart your Terminal (and in some cases your system) for the changes
 
 Next: [Configure for Mobile Targets](#configure-for-mobile-targets) if you'd like to build for Android and iOS. Otherwise [Create a Project](/guides/create/).
 
+## Node.js 
+
+:::note[JavaScript ecosystem]
+Only if you intend to use a JavaScript frontend framework 
+:::
+
+1. Download and install [Node LTS](https://nodejs.org)
```
Comment (dreyfus92) [inline src/content/docs/guides/prerequisites/index.mdx]:
```suggestion
1.  Go to [Node.js website](https://nodejs.org), download the Long Term Support (LTS) version and install it. 
```

File: src/content/docs/guides/prerequisites/index.mdx [hunk 3]
```diff
@@ -194,6 +194,28 @@ Be sure to restart your Terminal (and in some cases your system) for the changes
...
+2. Check if Node was succesfully installed by running:
+
+```sh
+node -v 
+# v20.10.0
+npm -v 
+# 10.2.3
+```
+
+Be sure to restart your Terminal (and in some cases your system) for the changes to take affect.
```
Comment (dreyfus92) [inline src/content/docs/guides/prerequisites/index.mdx]:
```suggestion
It's important to restart your Terminal to ensure it recognizes the new installation. In some cases, you might need to restart your computer.
```

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @vasfvitor, I left you a couple of nits, let me know what you think 😁

**Comment by vasfvitor** [comment]
@dreyfus92 I think now it's clearer :)

I added the "next" section, could you take a look? https://github.com/tauri-apps/tauri-docs/pull/1903/commits/3cbe43975fdcc6bfda3cf59580a2d6e3b04463ec

File: src/content/docs/guides/prerequisites/index.mdx [hunk 4]
```diff
@@ -192,7 +192,31 @@ Here is the file as a plain script: [rustup.sh](https://sh.rustup.rs/)
...
-Next: [Configure for Mobile Targets](#configure-for-mobile-targets) if you'd like to build for Android and iOS. Otherwise [Create a Project](/guides/create/).
+Next: [Configure for Mobile Targets](#configure-for-mobile-targets) if you'd like to build for Android and iOS, or, if you'd like to use a JavaScript framework, [install Node](#nodejs). Otherwise [Create a Project](/guides/create/).
+
+## Node.js 
+
+:::note[JavaScript ecosystem]
+Only if you intend to use a JavaScript frontend framework 
+:::
+
+1.  Go to [Node.js website](https://nodejs.org), download the Long Term Support (LTS) version and install it.
```
Comment (dreyfus92) [inline src/content/docs/guides/prerequisites/index.mdx]:
```suggestion
1. Go to [Node.js website](https://nodejs.org), download the Long Term Support (LTS) version and install it. 
```

**Review by dreyfus92 — APPROVED** [review]
This should do the work, thank you @vasfvitor 😁

**Pull request merged**